ajax/var_type.js

```diff
@@ -0,0 +1,23 @@
+const log = console.log,
+      name = `sanket`;
+var a = 5;
+let b = 10;
+//log(`using var answer ` = a); // Invalid left-hand side in assignment
+// when your string and variable print in console, string and variable between use operator 
+log(`using const answer ` + name); // sanket
```

Comment:
If you are using ES6 then use template strings, no need of concatenation operators.